products/coder.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Coder
+category: server-app
+iconSlug: coder
+permalink: /coder
+versionCommand: coder version
+releasePolicyLink: https://coder.com/docs/install/releases
+changelogTemplate: https://github.com/coder/coder/releases/tag/v__LATEST__
+releaseDateColumn: true
+eoasColumn: Active Support
```

Comment:
```suggestion
```

Already the default value.

Reply:
Without this, the column is hidden and does not show up.

Reply:
Oups, you're right, sorry.

Reply:
I wanted to add this column. I will submit a follow up PR.

Reply:
Done in #5590.